custom_components/localtuya/const.py

```diff
@@ -17,6 +17,13 @@
 CONF_OPEN_CMD = "open_cmd"
 CONF_CLOSE_CMD = "close_cmd"
 CONF_STOP_CMD = "stop_cmd"
+CONF_POSITIONING_MODE = "positioning_mode"
+CONF_CURRPOS = "currpos_dps"
+CONF_SETPOS = "setpos_dps"
+CONF_MODE_NONE = "none"
```

Comment:
These `CONF_MODE_` constants are not configuration options (I.e. they are not stored as "keys" in the configuration) and shall thus not be named differently. It's also not obvious where they belong. I suggest that you change it to `COVER_MODE_` instead and move them to `cover.py` as they aren't used anywhere else.